README.md

```diff
@@ -8,68 +8,147 @@ Bitwarden Passwordless.dev is a software toolkit that helps developers build FID
 
 Using Passwordless.dev means there's no need to read extensive W3C specification documentation, determine what cryptography to implement, or worry about managing stored public keys. The team behind Bitwarden will take care of that for you.
 
-The `passwordless-server` project contains the APIs, database, and other core infrastructure items needed for the backend of all passwordless clients. The server project is written in C# using .NET 7 with ASP.NET Core.
+The `passwordless-server` project contains the APIs, database, and other core infrastructure items needed for the backend of all passwordless clients.
 
-## Get started
+## Using Passwordless.dev
 
-### Use Passwordless.dev
+### Create an application
 
 To get started using Passwordless.dev:
 
 1. [Sign up](https://admin.passwordless.dev/signup/) for a free account here.
 2. [Create an application](https://docs.passwordless.dev/guide/get-started.html#create-an-application) in the admin console.
 3. Install the Passwordless.dev [JavaScript Client library](https://github.com/passwordless/passwordless-client-js).
-4. Start building registration and signin flows for your application. Refer to the [Passwordless.dev documentation](https://docs.passwordless.dev/) for help. Here are some basic examples to get you started:
+4. Start building registration and signin flows for your application. Refer to the [Passwordless.dev documentation](https://docs.passwordless.dev) for help.
 
-**Registration**:
+### Wire up your backend
 
-```js
-// Node.js - Code written for this step should run on your backend.
-
-const payload = {
-  "userId": "107fb578-9559-4540-a0e2-f82ad78852f7", // Required. A WebAuthn User Handle, which should be generated by your application. Max. 64 bytes.
-  "username": "pjfry@passwordless.dev", // Required. A human readable username used for user authentication, should be chosen by the user.
-  // ...For more options, please see the API reference for /register/token.
-};
-
-// POST the payload to the Passwordless.dev API using your API private secret.
-const apiUrl = "https://v4.passwordless.dev";
-const {token} = await fetch(apiUrl + "/register/token", {
-    method: "POST",
-    body: JSON.stringify(payload),
-    headers: {
-        "ApiSecret": "myapplication:secret:11f8dd7733744f2596f2a28544b5fbc4",
-        "Content-Type": "application/json"
+You can use Passwordless in conjunction with a variety of different backend platforms — see the [documentation](https://docs.passwordless.dev/guide/backend) for more info. Below is an example of a backend integration using ASP.NET Core and the [Passwordless SDK for .NET](https://github.com/bitwarden/passwordless-dotnet):
+
+```csharp
+// Add Passwordless to your service container
+services.AddPasswordlessSdk(options =>
+{
+    options.ApiSecret = "your_api_secret";
```

Comment:
You don't need both. Just the secret.

Reply:
The example in the docs sets up both... Should we update both?

Reply:
I can't find it in the docs, but if it is there, then it has to be removed